File: namsor/python/src/PhoneCodeExample.py
import json
from datetime import date, datetime
from pprint import pprint

from namsor_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"api_key": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.SocialApi(api_client).phone_code(
            first_name="Jamini",
            last_name="Roy",
            phone_number="09804201420",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling SocialApi#phone_code: %s\n" % e)

File: namsor/ruby/src/PhoneCodeExample.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["api_key"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::SocialApi.new.phone_code(
        "Jamini", # first_name
        "Roy", # last_name
        "09804201420", # phone_number
    )

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling SocialApi#phone_code: #{e}"
end
